src/app/frontend/logs/logstoolbar/logstoolbar.scss

```diff
@@ -14,6 +14,10 @@
 
 $logs-toolbar-color-black: #000;
 $logs-toolbar-color-white: #fff;
+$logs-toolbar-text-font-size: 17px;
+$logs-toolbar-title-font-weight: 500;
+$logs-toolbar-select-margin-bottom: 15px;
+$logs-toolbar-select-margin-left: auto;
```

Comment:
For 15px you could use 2*$baseline-grid. Will it look okay? The reason is that we should not use magic numbers unless really needed, and base everything on baseline grid.